db_stress_tool/db_stress_test_base.cc

```diff
@@ -2638,18 +2638,14 @@ void StressTest::Open(SharedState* shared) {
         db_aptr_.store(txn_db_, std::memory_order_release);
       }
 
-      // after a crash, rollback to commit recovered transactions
+      // After a crash, the uncommitted transaction will be rolled back after
+      // recovery.
       std::vector<Transaction*> trans;
       txn_db_->GetAllPreparedTransactions(&trans);
       Random rand(static_cast<uint32_t>(FLAGS_seed));
       for (auto txn : trans) {
-        if (rand.OneIn(2)) {
-          s = txn->Commit();
-          assert(s.ok());
-        } else {
-          s = txn->Rollback();
-          assert(s.ok());
-        }
+        s = txn->Rollback();
```

Comment:
I am a little concerned about test coverage decrease here. A prepared transaction can be either committed or rolled back upon next recovery.

Reply:
Okay -  I will look into Andrew's second suggestion "Alternatively, we could try deferring processing GetAllPreparedTransactions() until after the verification completes.".

Reply:
Fixed